docs/infrastructure-components.md

```diff
@@ -0,0 +1,164 @@
+# Infrastructure components
+
+We describe here the elements that form the infrastructure required for projects
+under KubeVirt's umbrella. This mainly involves components required to run our
+CI/CD systems and related services, like the monitoring stack or applications for
+indexing and querying build logs.
+
+The CI/CD system is based on [Prow], kubernetes CI system. For a brief overview
+of how Prow runs jobs take a look at ["Life of a Prow Job"]. To see common Prow
+usage and interactions flow, see the pull request interactions [sequence diagram].
+
+## Layout
+
+![infra-layout](infra-layout.png)
+
+## Clusters
+
+Our infrastructure currently uses 3 main clusters. For each of them we will
+include the following fields:
+
+* Description: brief explanation of the cluster purpose.
+* Context: context name of the cluster in the master kubeconfig used by Prow to
```

Comment:
```suggestion
* Prow context: context name of the cluster in the master kubeconfig used by Prow to
```
maybe?

Reply:
can you add a url to the master kubeconfig?

Reply:
Sure done. It is part of the automation secrets, though. There's  a good explanation about how it is handled in the document about adding new clusters proposed here #1487, added a link to it too.